### DISCUSSION TITLE
Screenshots for Project Ticket Layout settings

### DISCUSSION THREAD

**Discussion by brasofilo** [Leantime/leantime#500] [Q&A] [answered]
First of all, great project, thanks for pulling this up, Marcel, really appreciated!

I've been looking into every aspect of the interface and maybe I'm not seeing something obvious...  
What is the difference between Classic and Modern views? Can we have a couple of screenshots here on Discussions?

This settings appears when editing a project (`domain.com/projects/showProject/`) and we have this section **_Additional Settings_**

> ![image](https://user-images.githubusercontent.com/1542052/140264478-76b8c89c-7575-4adc-9edb-140dd3c7fe7e.png)

**Comment by marcelfolaron** [comment, marked as answer]
Hey, 
This feature setting was implemented as part of a ticket view redesign. Unfortunately the redesign is taking longer than anticipated so we have a stray setting that doesn't do much at the moment
